ChaosMod/Components/EffectDispatcher.cpp

```diff
@@ -226,6 +226,11 @@ EffectDispatcher::EffectDispatcher(const std::array<BYTE, 3> &timerColor, const
 	m_MetaEffectShortDur =
 	    g_OptionsManager.GetConfigValue<int>("MetaShortEffectDur", OPTION_DEFAULT_EFFECT_META_SHORT_TIMED_DUR);
 
+	m_EnableDistanceBasedEffectDispatch = g_OptionsManager.GetConfigValue<bool>("EnableDistanceBasedEffectDispatch", false);
+	m_DistanceToActivateEffect = g_OptionsManager.GetConfigValue<int>("DistanceToActivateEffect", 250);
+	m_DistanceType                      = static_cast<TravelledDistanceType>(
+        g_OptionsManager.GetConfigValue<int>("DistanceType", static_cast<int>(TravelledDistanceType::Distance)));
```

Comment:
The default value should be a def under Util/OptionsDefault.h for consistency with the other GetConfigValue calls

Reply:
👌